libkpod/config.go

```diff
@@ -0,0 +1,267 @@
+package libkpod
```

Comment:
This is not going to be a libkpod-specific thing - in fact, we may evolve a new config file for kpod itself at some point. It should stay in server